src/extension/module.js

```diff
@@ -2723,6 +2662,7 @@
     }
     if (userAction) {
       // log telemetry
+      /* c8 ignore next */
```

Comment:
What is this?

Reply:
This is to ignore the code coverage for the next line. We do not have a test for this line and the coverage drops because of it. I doubt we want to write tests for rum sampling...